pyocd/coresight/ap.py

```diff
@@ -798,7 +798,10 @@ def _init_rom_table_base() -> None:
             is_adiv5_base = (base & AP_BASE_FORMAT_MASK) != 0
             is_base_present = (base & AP_BASE_ENTRY_PRESENT_MASK) != 0
             is_legacy_base_present = not is_adiv5_base and not is_base_present
-            if is_legacy_base_present:
+            if base == 0xe00fe002:
```

Comment:
Any suggestions welcome on how to do the rom table lookup properly. Seems like we don't set that bit, but e.g. Segger doesn't seem to care about it, even without specific chip support.

Reply:
@ithinuel found and pushed a solution! :)